sdk/communication/azure-communication-identity/src/main/java/com/azure/communication/identity/models/CommunicationIdentityTokenScope.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.communication.identity.models;
+
+import com.azure.core.util.ExpandableStringEnum;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import java.util.Collection;
+
+/** Defines values for CommunicationIdentityTokenScope. */
+public final class CommunicationIdentityTokenScope extends ExpandableStringEnum<CommunicationIdentityTokenScope> {
```

Comment:
In Python we renamed this CommunicationTokenScope to be in sync with @RezaJooyandeh
which one should we do?

Reply:
Done, thanks Bert!